benchmark/function/camera_function.benchmark.cpp

```diff
@@ -0,0 +1,69 @@
+#include <benchmark/benchmark.h>
+
+#include <mbgl/style/function/source_function.hpp>
+
+#include <mbgl/style/rapidjson_conversion.hpp>
+#include <mbgl/style/conversion.hpp>
+#include <mbgl/style/conversion/function.hpp>
+
+#include <rapidjson/document.h>
+
+
+using namespace mbgl;
+using namespace mbgl::style;
+
+static rapidjson::GenericDocument<rapidjson::UTF8<>, rapidjson::CrtAllocator> createFunctionJSON(size_t stopCount) {
+    rapidjson::GenericDocument<rapidjson::UTF8<>, rapidjson::CrtAllocator> doc;
+
+    std::string stops = "[";
+    for (size_t i = 0; i < stopCount; i++) {
+        std::string value = std::to_string(24.0f / stopCount * i);
+        if (stops.size() > 1) stops += ",";
+        stops += "[" + value + ", " + value + "]";
+    }
+    stops += "]";
+    
+    doc.Parse<0>(R"({"type": "exponential", "base": 2,  "stops": )" + stops + R"(, "property": "x"})");
```

Comment:
`"property": "x"` looks like a copy and paste mistake from `source_function.benchmark.cpp`.

Reply:
Fixed